### DISCUSSION TITLE
Screen shot of shortcuts isn't correct...

### DISCUSSION THREAD

**Discussion by boxallw** [OnionUI/Onion#523] [Q&A]
The words are correct, but the picture is wrong for me at least:

https://github.com/jimgraygit/Onion#shortcuts

**Comment by Olywa** [comment]
As much as I can tell, the picture matches exactly what is described in the text (albeit with some additional shortcuts covered in the text)
What is it you think is incorrect in the screenshot?
Perhaps you remapped some buttons accidently (in settings-->Key mapping) or changed some shortcuts in Retroarch?

**Reply by boxallw** [reply]
Screen shot shows the select key as (Retroarch menu). You must push the menu key with select to get to the Retroarch menu.

**Reply by Artickw** [reply]
You need to hold the menu button to get any of the shortcuts described to work.

**Reply by Olywa** [reply]
@boxallw yes, _every_ shortcut there in the image requires you to press 'menu+' the key, to work  (which is I think is also what @Artickw is suggesting/confirming). 

The image is this way intentionally as a quick reference point. It would be very cluttered and unnecessary to label every single key with 'menu+' when this is stated already at the top of the image and the shortcuts are also explained below the image in detail).
